mapper/javabean/src/main/java/org/hibernate/search/mapper/javabean/log/impl/Log.java

```diff
@@ -34,4 +36,11 @@ SearchException cannotFindProperty(@FormatWith(PojoTypeModelFormatter.class) Poj
 	SearchException cannotReadProperty(@FormatWith(PojoTypeModelFormatter.class) PojoRawTypeModel<?> typeModel,
 			String propertyName);
 
+	@Message(id = ID_OFFSET_1 + 3,
+			value = "Exception while retrieving the type model for '%1$s'.")
+	SearchException errorRetrivingTypeModel(@FormatWith(ClassFormatter.class) Class<?> clazz, @Cause Exception cause);
+
+	@Message(id = ID_OFFSET_1 + 4,
+			value = "Exception while retrieving property type model for '%1$s' on '%2$s'")
+	SearchException errorRetrivingPropertyTypeModel(String propertyModelName, String parentTypeModel, @Cause Exception cause);
```

Comment:
Also, you could pass the type model directly instead of using toString(). Just use a formatter, like on `org.hibernate.search.mapper.pojo.logging.impl.Log#unableToResolveDefaultIdentifierBridgeFromSourceType` for example.